scripts/develop

```diff
@@ -61,11 +48,42 @@ wp.on('aggregated', () => {
     return;
   }
 
+  console.log('\nBuilding web-ext webpack bundle');
+  const webpackSuccess = await webpack().then(() => true, () => false);
+
+  if (!webpackSuccess) {
+    notify('webpack build errors');
+    return;
+  }
+
   console.log('\nRunning functional tests');
   if (!mochaFunctional()) {
     notify('mocha functional tests errors');
     return;
   }
+}
+
+wp.on('change', (changedFile, mtime) => {
+  if (mtime == null) {
```

Comment:
Let's use `=== null` to avoid accepting `undefined`. The documentation at https://github.com/webpack/watchpack/tree/9e8152b850a2492cb27f624e8a898ea23fde199f#api only says something about `null`, not `undefined`.

Reply:
sounds good, tweak applied in e3fc21e